src/gengowatcher/website_setup.py

```
@@ -0,0 +1,221 @@
+    config.save_config()
+
+    log("\n" + "=" * 60)
+    log("    Setup Complete!")
+    log("=" * 60)
+    log("")
+    log("Configuration saved to config.ini:")
+    log(f"  Jobs URL:        {jobs_url}")
+    log(f"  Session Cookie:  {'*' * 20} (hidden)")
+    log(f"  Headless Mode:   {'Yes' if headless else 'No'}")
+    log(f"  Check Interval:  {check_min}-{check_max} seconds")
+    log(f"  Enabled:         Yes")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)